packages/cosmos_ui_components/lib/components/cosmos_image_button.dart

```diff
@@ -21,7 +22,7 @@ class CosmosImageButton extends StatelessWidget {
       child: SizedBox(
         height: height,
         width: width,
-        child: Image.asset(asset),
+        child: Image.asset(asset, package: packageName),
```

Comment:
this won't work in this particular example. if someone tries to use the `CosmosImageButton` with an asset outside of the `cosmos_ui_components`. what you should do here is to allow specify the package as a constructor param that defaults to null and provide the packageName at the call site

Reply:
Updated

Reply:
you now also have to update call sites (i.e: CosmosWalletsListView) to pass in the proper `package`

Reply:
Updated